### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_EditMetada_Keyboard: Keyboard Trap occurs inside the text area of the Popup.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4005]
**Environment Details:**
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 68.0.3440.106 
Operating System:Windows10

**Steps to reproduce:**
1.Login with the appropriate credentials to azure notebook account.    

2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library

3.Click on Run or right click and select Run to run the notebook.
4.Navigate to any file present under Files tab-->Hit enter-->View Menu-->Cell Toolbar-->Edit Metadata. Click on Edit Metadata button appeared on page.
5. Try to navigate throughout the text box using Keyboard.

**Actual**
Keyboard trap occurs inside the text area of the Popup. User can't get out of the text area using TAB or 'Down' arrow key.

This issue exist for 'Chrome + NVDA' configuration also.

**Expected**
Keyboard User should be able to navigate through text-box using TAB or any other keys.

**User Impact**
Keyboard only user will get struck inside the text box and will not be able to reach to other controls of the popup using the keyboard.
**Attachment:**
[A11y_AzureNotebooks_Jupyter_EditMetada_Keyboard18.pptx](https://github.com/jupyter/notebook/files/2401152/A11y_AzureNotebooks_Jupyter_EditMetada_Keyboard18.pptx)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
